Sources/PipeableSDK/Script/Fakes.swift

```diff
@@ -0,0 +1,98 @@
+import Foundation
+import JavaScriptCore
+
+// Fakes are temporary used for testing. Will be removed soon.
+
+class FakePage {
+    func goto(_ url: String) async throws {
+        if url.isEmpty {
+            throw PipeableError.navigationError("Empty url")
+        }
+        print("Navigating to url: \(url) ...")
+        try await Task.sleep(nanoseconds: 1_000_000_000)
+        print("Navigation completed!")
+    }
+
+    public func querySelector(_ selector: String) async throws -> FakePipeableElement? {
+        if selector.isEmpty {
+            return nil
+        }
+        if selector == "empty" {
+            return FakePipeableElement(self, "")
+        }
+        return FakePipeableElement(self, "id-123")
+    }
+}
+
+class FakePipeableElement {
+    var page: FakePage
+    let elementId: String
+
+    init(_ page: FakePage, _ elementId: String) {
+        self.page = page
+        self.elementId = elementId
+    }
+
+    public func click() async throws {
+        if elementId.isEmpty {
+            throw PipeableError.elementNotFound
+        }
+        print("Clicking on element with id: \(elementId) ...")
+        try await Task.sleep(nanoseconds: 1_000_000_000)
+        print("Click completed!")
+    }
+}
+
+@objc protocol FakeElementJSExport: JSExport {
+    func clickWithCompletion(_ completionHandler: JSValue)
+}
+
+class FakeElementWrapper: BaseWrapper, FakeElementJSExport {
+    let element: FakePipeableElement
+    var successFullClicksClount = 0
+
+    init(_ dispatchGroup: DispatchGroup, _ element: FakePipeableElement) {
+        self.element = element
+        super.init(dispatchGroup)
+    }
+
+    func clickWithCompletion(_ completionHandler: JSValue) {
+        doWithCompletion(completionHandler) {
+            try await self.element.click()
+            self.successFullClicksClount += 1
+            return CompletionResult.success(nil)
+        }
+    }
+}
+
+@objc protocol FakePageJSExport: JSExport {
```

Comment:
Same as above

Reply:
Those will be deleted soon. The goal is not to keep them in sync with the real one.